src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -128,7 +130,7 @@ ApiGlobe.prototype.addImageryLayerFromJSON = function addImageryLayerFromJSON(ur
 
 /**
  * This function adds an imagery layer to the scene using an array of JSON files. The layer id must be unique. The protocol rules wich parameters are then needed for the function.
- * @constructor
+ * <iframe width="100%" height="400" src="//jsfiddle.net/iTownsIGN/763trrk8/embedded/" allowfullscreen="allowfullscreen" frameborder="0"></iframe>
```

Comment:
`https` won't work as the itowns lib is loaded in http in JSFiddle...
But the `HTTP` protocol works fine, so I can put 'http' on every URLs. WDYT ?